src/otx/core/metrics/kitti_3d_eval/eval.py

```diff
@@ -0,0 +1,811 @@
+# flake8: noqa
+# mypy: ignore-errors
```

Comment:
It should be located under docstring, isn't it?

Reply:
done